app/views/layouts/turbo_rails/frame.html.erb

```diff
@@ -0,0 +1 @@
+<html><head><%= yield :head %></head><body><%= yield %></body></html>
```

Comment:
If we're going to be introducing this template, what do you think about formatting its source with line breaks for the sake of legibility? I think the savings on omitting the newline character might not be pulling their weight.

```erb
<html>
  <head><%= yield :head %></head>
  <body><%= yield %>
</html>
```

Reply:
Yeah, that's a good idea 👍